src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -367,6 +367,12 @@ def conn_wrapper(self, fs_handle, **kwargs):
             return result
         return conn_wrapper
 
+    def nametojson(self, names):
+        dict = []
+        for i in range(len(names)):
+            dict.append({'name': names[i]})
+        return 0, json.dumps(dict, indent=2), ""
```

Comment:
I'd prefer if this helper just return `json.dumps(dict, indent=2)`

Reply:
Updated